clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/HubConnection.java

```diff
@@ -406,4 +410,8 @@ public void remove(String name) {
         handlers.remove(name);
         logger.log(LogLevel.Trace, "Removing handlers for client method %s" , name);
     }
+
+    public void onClosed(Action1<Exception> callback) {
+        this.onClosedCallback = callback;
```

Comment:
Should we support multiple listeners? TypeScript and .NET do.

Reply:
Yeah, you're right

Reply:
Curious to know if anyone actually registers more than one OnClosed handlers though